site/content/en/docs/Guides/troubleshooting.md

```diff
@@ -5,6 +5,193 @@ weight: 200
+    State:          Running
+      Started:      Fri, 16 Aug 2019 21:25:48 +0000
+    Ready:          True
+    Restart Count:  0
+    Requests:
+      cpu:     30m
+    Liveness:  http-get http://:8080/healthz delay=3s timeout=1s period=3s #success=1 #failure=3
+    Environment:
+      GAMESERVER_NAME:  simple-udp-zqppv
+      POD_NAMESPACE:    default (v1:metadata.namespace)
+    Mounts:
+      /var/run/secrets/kubernetes.io/serviceaccount from agones-sdk-token-vr6qq (ro)
+Conditions:
+  Type              Status
+  Initialized       True
+  Ready             True
+  ContainersReady   True
+  PodScheduled      True
+Volumes:
+  empty:
+    Type:    EmptyDir (a temporary directory that shares a pod's lifetime)
+    Medium:
+  agones-sdk-token-vr6qq:
+    Type:        Secret (a volume populated by a Secret)
+    SecretName:  agones-sdk-token-vr6qq
+    Optional:    false
+QoS Class:       Burstable
+Node-Selectors:  <none>
+Tolerations:     node.kubernetes.io/not-ready:NoExecute for 300s
+                 node.kubernetes.io/unreachable:NoExecute for 300s
+Events:
+  Type    Reason     Age                   From                                             Message
+  ----    ------     ----                  ----                                             -------
+  Normal  Scheduled  2m32s                 default-scheduler                                Successfully assigned default/simple-udp-zqppv to gke-test-cluster-default-590db5e4-4s6r
+  Normal  Pulling    2m31s                 kubelet, gke-test-cluster-default-590db5e4-4s6r  pulling image "gcr.io/agones-mark/agones-sdk:1.0.0-dce1546"
+  Normal  Started    2m28s                 kubelet, gke-test-cluster-default-590db5e4-4s6r  Started container
+  Normal  Pulled     2m28s                 kubelet, gke-test-cluster-default-590db5e4-4s6r  Successfully pulled image "gcr.io/agones-mark/agones-sdk:1.0.0-dce1546"
+  Normal  Created    2m28s                 kubelet, gke-test-cluster-default-590db5e4-4s6r  Created container
+  Normal  Created    114s (x2 over 2m31s)  kubelet, gke-test-cluster-default-590db5e4-4s6r  Created container
+  Normal  Started    114s (x2 over 2m31s)  kubelet, gke-test-cluster-default-590db5e4-4s6r  Started container
+  Normal  Pulled     114s (x2 over 2m31s)  kubelet, gke-test-cluster-default-590db5e4-4s6r  Container image "gcr.io/agones-images/udp-server:0.14" already present on machine
+```
+
+Finally, you can also get the logs of your `GameServer` `Pod` as well via `kubectl logs <pod name> -c <game server container name>`, for example:
+
+```
+root@6a71afd42291:/go/src/agones.dev/agones# kubectl logs simple-udp-zqppv -c simple-udp
+2019/08/16 21:26:23 Creating SDK instance
+2019/08/16 21:26:24 Starting Health Ping
+2019/08/16 21:26:24 Starting UDP server, listening on port 7654
+2019/08/16 21:26:24 Marking this server as ready
+```
+
+Do note that this will only give the most recent container's logs (so we won't get the previous crash). Usually this is
```

Comment:
If you run `kubectl logs -p simple-udp-zqppv -c simple-udp` it will give you the logs from the prior execution of the container so that you can see the previous crash.

Reply:
Wait. You can do what now? _runs off to try_ 😮

Reply:
Done!